python/bids.py

```diff
@@ -15,14 +16,9 @@ class Bid(sdk.python.entities.Entity):
     resource = '/bids/'
     json_name = 'bid'
 
-    def __init__(self):
-        super(Bid, self).__init__()
-
-        self.json_property(int, 'id')
-        self.json_property(datetime.datetime, 'agreed_deadline')
-        self.recursive_json_property(sdk.python.jobs.Assignment,
-                                     "assignments")
-        self.recursive_json_property(BidItem, 'bid_items')
+    id = Property(int)
+    agreed_deadline = Property(datetime.datetime)
+    bid_items = Property(BidItem)
```

Comment:
is this on purpose that assignments are not include?

Reply:
that's given as a backref from Assignment. (can't promise that i got every single backref right, though)